### ISSUE TITLE
Bollinger Bands don't work as desired 

### ISSUE THREAD

**Issue by swampibboy** [GimmerBot/Gimmer#38]
Linux chrx 4.9.4-galliumos-braswell #1 SMP PREEMPT galliumos2 Thu Feb 23 01:58:04 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux

Order type
market
Exchange
Binance
Base
bnb
Quote
Usdt
time interval
6 hr
From
02/16/2018 02:10
To
05/18/2018 02:10

bollinger bands
default setting

![image](https://user-images.githubusercontent.com/30175655/40227414-f4e56dbe-5a7d-11e8-8a7b-2c51f567a955.png)

I can not work out what is happening here , over the time frame that the test was run there was one opportunity for a buy / sell order, there are literally dozens of trades on this chart and not one of them makes any trading sense.

**Comment by Flllexa** [comment]
Yes, I think that I know how can I improved that, sorry man, I implement very fast for publish included in beta 👍

**Comment by swampibboy** [comment]
no need to hurry :) i know you guys are working around the clock to bring us beta update :)...this is just so strange  that i can not see what is happening here !!

**Comment by keithfawcett** [comment]
BB is still broken. Although we're half way there. We are not buying/selling outside of the band, but the buys and sells are backwards and require the selection the "Invert Signal" checkbox to work correctly. Please reverse the buys/sells so we do not have to select the "Invert Signal" for normal operation. 

![image](https://user-images.githubusercontent.com/592055/40802322-48fc0132-64e3-11e8-93b9-379150e30645.png)
